src/quo2/components/messages/system_message.cljs

```diff
@@ -0,0 +1,145 @@
+(ns quo2.components.messages.system-message
+  (:require [status-im.i18n.i18n :as i18n]
+            [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [quo2.components.buttons.button :as button]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.avatars.user-avatar :as user-avatar]
+            [quo2.components.avatars.icon-avatar :as icon-avatar]))
+
+(def themes
+  {:light {:text       colors/black
+           :icon       colors/primary-50
+           :time       colors/neutral-50
+           :background colors/neutral-5}
+   :dark  {:text       colors/white
+           :icon       colors/primary-50
+           :time       colors/neutral-40
+           :background colors/neutral-95}})
+
+(defn get-color [key]
+  (get-in themes [(theme/get-theme) key]))
+
+(defn sm-timestamp [timestamp-str]
+  [rn/view {:margin-left 6}
+   [text/text {:size  :label
+               :style {:color          (get-color :time)
+                       :text-transform :none}}
+    timestamp-str]])
+
+(defn sm-icon [icon]
+  [rn/view {:align-items  :center
+            :margin-right 8}
+   [icon-avatar/icon-avatar {:size  :large
+                             :icon  icon
+                             :color :primary
+                             :opacity 5}]])
+
+(defmulti sm-render :type)
+
+(defmethod sm-render :deleted [_]
+  [rn/view {:align-items     :center
+            :justify-content :space-between
+            :flex            1
+            :flex-direction  :row}
+   [rn/view {:align-items    :center
+             :flex-direction :row}
+    [sm-icon :main-icons/placeholder20]
+    [text/text {:size  :paragraph-2
+                :style {:color        (get-color :text)
+                        :margin-right 5}}
+     (i18n/label :message-deleted-for-everyone)]]
+   [button/button {:size   24
+                   :before :main-icons/timeout
+                   :type   :grey} (i18n/label :undo)]])
```

Comment:
It seems the `deleted` system message is actually here https://www.figma.com/file/PPWkgOYlZZDxZv5SDGsZVV/Posts-%26-Attachments?node-id=4278%3A512149 
no undo button, and it might need to fade the color on timeout https://discord.com/channels/624205794384281629/626744384565674034/1024233617762811944

We can, of course, implement that with a seperate PR

Reply:
Done